libs/ml/tests/unit/ops/activations/softmax.cpp

```diff
@@ -19,10 +19,9 @@
 #include "core/serializers/main_serializer_definition.hpp"
 #include "gtest/gtest.h"
 #include "math/base_types.hpp"
-#include "ml/ops/activation.hpp"
+#include "ml/ops/activations/softmax.hpp"
```

Comment:
Do I understand right that this change just removes unnecessary include? I mean, activation.hpp only includes softmax.hpp, and it is the softmax.hpp that we really need here? [good change!]

Reply:
activation.cpp includes all the activations; here we only need softmax. Plus, the activation.hpp logic has been moved into source